src/components/Asset/AssetActions/Pool/Remove/_utils.ts

```diff
@@ -1,19 +1,42 @@
-import { calcMaxExactOut } from '@oceanprotocol/lib'
+import { PoolInfo, PoolInfoUser } from '@context/Pool/_types'
+import { calcMaxExactOut, Pool } from '@oceanprotocol/lib'
 import Decimal from 'decimal.js'
+// eslint-disable-next-line camelcase
+import { PoolData_poolData } from 'src/@types/subgraph/PoolData'
```

Comment:
in other places when using those subgraph typings we convert this to a camelcase typename which then ESLint seem to like again. So in this case maybe:

```suggestion
import { PoolData_poolData as PoolData } from 'src/@types/subgraph/PoolData'
```